### PR TITLE
Add Cloze

### PR DESCRIPTION
**Pull request by claudinec**
Alexa global rank: 68,258.

### CODE REVIEW

**Review by kmpoppe — CHANGES_REQUESTED** [review]
Hello @claudinec!
The image you supplied is larger than 2500 bytes, which is not allowed under the current rules. Please use https://tinypng.com/ to reduce the size of the image and change it in your branch by a new checkin.
Thanks for your contribution!

**Comment by claudinec** [comment]
Oops, missed that, try now?

**Review by kmpoppe — APPROVED** [review]

**Review by kmpoppe — DISMISSED** [review]

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
You've defined them as supporting 2FA without any 2FA tags. Please add either `software`, `hardware`, `email`, `sms` or `phone`.

File: _data/communication.yml [hunk 1]
```diff
@@ -34,6 +34,12 @@ websites:
       twitter: CarbonBlack_Inc
       facebook: carbonblackinc
 
+    - name: Cloze
+      url: https://www.cloze.com/
+      img: cloze.png
+      tfa: Yes
+      doc: https://help.cloze.com/knowledge_base/topics/how-do-i-enable-2-step-authentication
```
Comment (RasmusKallas) [inline _data/communication.yml]:
```suggestion
      doc: https://help.cloze.com/knowledge_base/topics/how-do-i-enable-2-step-authentication
      sms: Yes
```

**Review by jtagcat — CHANGES_REQUESTED** [review]
Cloze supports only sms.

**Comment by claudinec** [comment]
Sorry, I should have checked this at the start.

The Cloze documentation doesn't match the reality - it offers both SMS and app-based authentication methods, as in the screenshot. I have emailed Cloze support asking them to update their documentation.

cloze-2019-04-16

**Comment by claudinec** [comment]
Cloze have updated their documentation: https://help.cloze.com/knowledge_base/topics/how-do-i-enable-2-step-authentication

File: _data/communication.yml [hunk 2]
```diff
@@ -34,6 +34,12 @@ websites:
       twitter: CarbonBlack_Inc
       facebook: carbonblackinc
 
+    - name: Cloze
+      url: https://www.cloze.com/
+      img: cloze.png
+      tfa: Yes
```
Comment (kmpoppe) [inline _data/communication.yml:40]:
```suggestion
      tfa: Yes
      sms: Yes
      software: Yes
```

File: _data/communication.yml (hunk 1 above)
Comment (kmpoppe) [inline _data/communication.yml:43]: @vc7ee I've now added sms and software (as per the new documentation) to the suggested change, would you please remove this suggestion?

**Review by jtagcat — DISMISSED** [review]

**Comment by Carlgo11** [comment]
@kmpoppe I can't approve the PR before the required changes have been implemented.

**Comment by kmpoppe** [comment]
@Carlgo11 Sorry, I thought I had committed the changes, but I didn't ... 🤔
@claudinec Would you please incorporate my suggestion into your files?

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**